test/isolation/specs/deadlock_insert_select.spec

```diff
@@ -0,0 +1,27 @@
+##github issue 865.
```

Comment:
shouldn't this file be called deadlock_drop_chunks_select.spec. I don't see why the original name has `insert` in it. Also please expand the top comment of the file to say what you are testing -- that deadlocks don't occur between selects and inserts.